styles/base.less

```diff
@@ -466,3 +466,16 @@ atom-text-editor.is-focused .line.cursor-line {
   font-style: italic;
   color: #ffc600;
 }
+
+/*Nested elements in JSX*/
+.JSXNested {
+  color: #F8F8F8;
+}
```

Comment:
Due to Atom 1.13, the selector here should be `.syntax--JSXNested`.

Additionally, when looking at HTML in the Atom version and JSX/HTML in the Sublime version, nested elements are pure white. For consistency, I think the color here should be `#FFFFFF`.